tests/static_multimap/static_multimap_test.cu

```diff
@@ -0,0 +1,443 @@
+/*
```

Comment:
We should have tests for a custom key type with corresponding custom equality and hash functions, e.g., https://github.com/NVIDIA/cuCollections/blob/0d602ae21ea4f38d23ed816aa948453d97b2ee4e/tests/static_map/static_map_test.cu#L81-L111